src/Command/PullRequest/PullRequestMergeCommand.php

```diff
@@ -103,6 +106,20 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $prType = $input->getArgument('pr_type');
         $squash = $input->getOption('squash') || $input->getOption('force-squash');
 
+        /** @var Config $config */
+//        $config = $this->getApplication()->getConfig();
```

Comment:
You should still validate its validness when the value is passed an argument to prevent typos ;)

And you can move this logic to a private method to prevent introducing to much complexity.